website/docs/set-up-your-environment.mdx

```diff
@@ -0,0 +1,132 @@
+import Tabs from  '@theme/Tabs'
+import TabItem from  '@theme/TabItem'
+
+# Set up your environment
+The first step to build with ComposeDB is setting up your development environment. This guide will show you how to install and configure relevant packages and tools.
+
+## Installation
+### Create your project directory
+Start with creating the project directory. Here you’ll store all your app’s local files:
+```sh
+mkdir my-project
+cd my-project
+```
+
+### Node.js
+If you don’t already have them installed, you will need to install Node.js v16 and the following package managers:
+- [NodeJS v16](https://nodejs.org/en/)
+- [pnpm v7](https://pnpm.io/installation)
```

Comment:
Why do we force please to use pnpm please? We used to have tabs for pnpm, npm and yarn that would likely be more useful for readers

Reply:
i'll ask justina